ui/resizable.js

```diff
@@ -1040,14 +1028,7 @@ $.ui.plugin.add("resizable", "alsoResize", {
 					el.css(style);
 				});
 			};
-
-		if (typeof(o.alsoResize) === "object" && !o.alsoResize.nodeType) {
-			$.each(o.alsoResize, function(exp, c) {
-				_alsoResize(exp, c);
-			});
-		} else {
-			_alsoResize(o.alsoResize);
-		}
+		_alsoResize(o.alsoResize);
```

Comment:
The function call is no longer needed since there's a single path. Just do the looping directly. The logic inside the loop should also be cleaned up since there are no longer any contexts provided for the searching.